prow/gerrit/adapter/adapter.go

```diff
@@ -233,18 +232,25 @@ func (c *Controller) syncChange(latest client.LastSyncState, changeChan <-chan C
 // Sync looks for newly made gerrit changes
 // and creates prowjobs according to specs
 func (c *Controller) Sync() {
-	processSingleInstance := func(instance string) {
+	processSingleProject := func(instance, project string) {
 		// Assumes the passed in instance was already normalized with https:// prefix.
-		log := logrus.WithField("host", instance)
-		syncTime := c.tracker.Current()
-		latest := syncTime.DeepCopy()
+		log := logrus.WithFields(logrus.Fields{"host": instance, "repo": project})
+		tracker := c.tracker.Current()
+		syncTime := time.Now()
+		if projects, ok := tracker[instance]; ok {
+			if t, ok := projects[project]; ok {
+				syncTime = t
+			}
+		}
+		latest := tracker.DeepCopy()
 
 		now := time.Now()
 		defer func() {
 			gerritMetrics.changeProcessDuration.WithLabelValues(instance).Observe(float64(time.Since(now).Seconds()))
 		}()
 
-		changes := c.gc.QueryChangesForInstance(instance, syncTime, c.config().Gerrit.RateLimit)
+		// Ignore the error. It is already logged.
+		changes, _ := c.gc.QueryChangesForProject(instance, project, syncTime, c.config().Gerrit.RateLimit)
 		timeQueryChangesForInstance := time.Now()
```

Comment:
Shouldn't this be renamed to `timeQueryChangesForProject`? The log line just below should probably also be updated similarly.